File: perl/src/tests/t_write.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$result = Atrshmlog::attach();

$ret = Atrshmlog::write(1, $Atrshmlog::EVENT_POINT_IN_TIME_C, 42, 0, 0, "hallo");

print "write : $ret : \n";

print " \n";

exit (0);

# end of main

File: python/src/tests/t_write.py
#!/usr/bin/python3
#
# Id:
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

result = atrshmlog.attach()

ret = atrshmlog.write(1, atrshmlog.EVENT_POINT_IN_TIME_C, 42, 0, 0, "hallo")

print('write : ' + str(ret) + ' : ')

print (' ')

exit(0)

# end of test

